src/environmentd/tests/sql.rs

```diff
@@ -1381,12 +1380,6 @@ fn test_linearizability() -> Result<(), Box<dyn Error>> {
     // equal timestamp in strict serializable mode.
     assert!(join_ts >= view_ts);
 
-    mz_client.batch_execute(&"SET transaction_isolation = serializable")?;
```

Comment:
Does this no longer pass?

Reply:
This no longer passes since the table exists at the time of the view query it will push the timestamp of the view query back to the `upper - 1` of the table. Then the join will also be at `upper - 1` of the table.

In the above serializable assert, we make sure to query the view before the table exists. So the timestamp of the view query is `upper - 1` of the view.